### PR TITLE
Adding equolver

### PR DESCRIPTION
**Pull request by gigjozsa**
As it says, this adds equolver to Stimela. An equolver image is already uploaded to quay.io/stimela. I can provide a test file if wished.

### CODE REVIEW

File: stimela/cargo/base/equolver/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM stimela/base:1.2.5
```
Comment (o-smirnov) [inline stimela/cargo/base/equolver/Dockerfile:1]: Any particular reason for such an old base, @gigjozsa? Current is 1.6.0.

**Review by o-smirnov — COMMENTED** [review]
ok to test

File: stimela/cargo/base/equolver/Dockerfile (hunk 1 above)
Comment (gigjozsa) [reply stimela/cargo/base/equolver/Dockerfile:1]: Nope. I will update. And just found out why keeping the tags synchronized with the version is not so simple.

**Comment by o-smirnov** [comment]
retest this please

**Review by o-smirnov — APPROVED** [review]

**Pull request merged**